db/compaction/compaction.h

```diff
@@ -435,13 +435,13 @@ class Compaction {
                                       const int start_level,
                                       const int output_level);
 
+  // mark (or clear) all files that are being compacted
+  void MarkFilesBeingCompacted(bool mark_as_compacted);
```

Comment:
nit not relevant to your changes: mark_as_compacted -> mark_as_being_compacted

Reply:
done